src/evm_transition_tool/transition_tool_methods/_filesystem.py

```diff
@@ -0,0 +1,40 @@
+import os
```

Comment:
I think it's a great idea to move these generic helper utils to a separate module, but I'd skip the sub-directory and remove the underscore `_`, and call it something like
`src/evm_transition_tool/utils.py` or `src/evm_transition_tool/file_utils.py`

Perhaps we can put it somewhere more neutral, but we can worry about that later.